templates/bake/job.twig

```diff
@@ -18,21 +18,22 @@ declare(strict_types=1);
 
 namespace {{ namespace }}\Job;
 
+use Cake\Queue\Job\JobInterface;
 use Cake\Queue\Job\Message;
 use Interop\Queue\Processor;
 
 /**
  * {{ name }} job
  */
-class {{ name }}Job
+class {{ name }}Job implements JobInterface
 {
     /**
      * Executes logic for {{ name }}Job
      *
      * @param \Cake\Queue\Job\Message $message job message
      * @return string
```

Comment:
This should be `string|null` to match.

Reply:
I can update that. Should I update the doc blocks in the [MailerJob](https://github.com/cakephp/queue/blob/master/src/Job/MailerJob.php#L32-L34) as well?